packages/ui/src/validators/modals/validatorCard/ValidatorDetail.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { ButtonPrimary } from '@/common/components/buttons'
+import { MarkdownPreview } from '@/common/components/MarkdownPreview'
+import { ModalFooter } from '@/common/components/Modal'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { SidePaneBody, SidePaneLabel, SidePaneRow, SidePaneText } from '@/common/components/SidePane'
+import { NumericValueStat, StatisticsThreeColumns } from '@/common/components/statistics'
+import { TextSmall } from '@/common/components/typography'
+
+import { Validator } from '../../types'
+
+interface Props {
+  validator: Validator
+}
+
+export const ValidatorDetail = ({ validator }: Props) => {
+  return (
+    <>
+      <SidePaneBody>
+        <Details gap={24}>
+          <RowGapBlock gap={4}>
+            <h6>Key elements</h6>
+            <ModalStatistics>
+              <Stat size="s" value={20000}>
+                <TextSmall lighter>Total reward</TextSmall>
+              </Stat>
+              <Stat size="s" value={validator.isVerified ? 'Verified' : 'Unverified'}>
+                <TextSmall lighter>Status</TextSmall>
+              </Stat>
+              <Stat size="s" value={validator.slashed.toString()}>
```

Comment:
This should have "times" or "time" at the end.

You could use:
```suggestion
              <Stat size="s" value={`${validator.slashed} time${plural(validator.slashed)}`}>
```